snykScripts/scan.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+# shellcheck source=scripts/lib.sh
+source "$(dirname "$0")/lib.sh"
+
+# Scan project
+
+# Environment prep/validation
+ecr_login
+if test -n "$JENKINS_HOME"; then
+  # Jenkins build concerns
+  require_environment_variable "JENKINS_HOME" "$JENKINS_HOME"
+  require_environment_variable "SNYK_TOKEN", "$SNYK_TOKEN"
+else
+  echo "Not yet supported outside of Jenkins"
+  exit 1
+fi
+
+deployment_id="1"
```

Comment:
Maybe it's worth commenting here in the script that these `deployment_{property}` vars are just being set to dummy values to appease some of the gradle build requirements (for the compose prototype IIRC?).